### ISSUE TITLE
Pep + absolutely positioned elements = disaster

### ISSUE THREAD

**Issue by ScottBeeson** [briangonzalez/jquery.pep.js#203]
Has anyone else experienced this?  Here is a Stack Overflow question about it: http://stackoverflow.com/questions/44075838/pep-js-constrain-to-parent-breaks-absolutely-positioned-elements

**Comment by ScottBeeson** [comment]
If you position absolutely using `left` instead of `right` it seems to work fine.

**Comment by seanCodes** [comment]
Looks like that SO question was moved... 😕 Do you have a jsfiddle or codepen link demoing the issue?

**Comment by ScottBeeson** [comment]
No, but it's easily reproducable. Any element that is absolutely positioned using `right: ##px` will break because pep calculates everything from the left.
